tests/default_vars.sh

```diff
@@ -1799,6 +2125,7 @@ export_hafs ()
 {
 export_fv3_v16
 export FV3=true
+export MPAS=false
```

Comment:
If FV3 OFF by default in the top level cmake file. Why it is true by default in here. So, it would be nice to make them consistent.